content/docs/coldchain/equipment.md

```diff
@@ -0,0 +1,205 @@
++++
+title = "Equipment"
+description = "Management of Cold Chain Equipment"
+date = 2023-11-08T15:20:00+00:00
+updated = 2023-11-08T15:20:00+00:00
+draft = false
+weight = 1
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+toc = true
+top = false
++++
+
+Cold Chain Equipment is a digital register fpr the creation and management of cold chain assets. It allows health departments to manage, record functional status logs, and update locations of cold chain assets.
+
+From the **Equipment** menu, all assets available in a store can be viewed.
+
+All cold chain assets of all stores are visible when accessing the **Equipment** menu via Open mSupply central server.
```

Comment:
Would like this central server text linked to the central server page when it's up